packages/react-native-sdk/README.md

```diff
@@ -1,30 +1,11 @@
 # Kinvey React Native SDK
 
-## Installation
+The Kinvey React Native SDK is used to develop an React Native application that connects to Kinvey.
 
-From the command prompt go to your app's root folder and execute:
+## Install
```

Comment:
What happened to the rest of this README, which describes the current state of the shim and how to use the demo app?

Reply:
I added back the missing sections in d8ab435.